docs/DeveloperGuide.md

```diff
@@ -375,16 +375,16 @@ The main logic for `find` command is still implemented within `FindCommand` (whi
 
 The `FindCommandParser` class has been extended to include a switch-case block to validate
 which field the user is trying to search across:
-1. `studentId`
+1. `studentId` (accepted user input abbreviated as `id`)
 2. `course`
```

Comment:
Would we need this?

```suggestion
2. `course` (accepted user input abbreviated as `c`)
```

Reply:
not relevant here! Have kept the user input for find command as `course`! will update if we deem it necessary to shorten `course` to `c` as well?

Reply:
For `add` and `edit`, both are using `c` char

Reply:
this section is for the `find` command! not the `edit` and `add`! Hmmm, unless your suggestion is to change `course` to `c`, `phone` to `p` and `email` to `e` as well for the `find` command?

Reply:
hey @charmainehly, yup i know it is for that. i'm okay either ways. i just thought of the consistency between all the commands that's all. tbh, if no t ime, let's let it go. 👍🏻 i'll let you decide boss!